### ISSUE TITLE
Perhaps the static location hook should support base path by default

### ISSUE THREAD

**Issue by roj1512** [molefrog/wouter#266]

**Comment by molefrog** [comment]
That could actually be a good idea. This sounds like a nice first issue for someone who would like to contribute.
To clarify this is how it is supposed to work:

```js
<Router hook={staticLocation("/page-to-render")} base="/app">
   <Route path="/page-to-render"> {/* this will be rendered, matches static location */}
     <Link to="/another-page" />{/* this rendered link will have href="/app/another-page" *}
  </Route>
</Router>
```

**Comment by roj1512** [comment]
I wish you had Deno-first code in the root so we could directly use it, even before releases :)

**Comment by roj1512** [comment]
@molefrog I don't think there's anything wrong other than the advantages if you make a release now.

**Comment by molefrog** [comment]
> @molefrog I don't think there's anything wrong other than the advantages if you make a release now.

Fair enough. Released in v2.9.0

**Comment by roj1512** [comment]
Thanks!
